sites/istio.io/README.md

```diff
@@ -0,0 +1,120 @@
+Overview
+====
```

Comment:
You need to update that file with current setup. There is not more make test and only one deployment (prod).

Reply:
Alternatively you can just delete it.